_posts/2018-03-01-core-team-announcement.md

```diff
@@ -0,0 +1,34 @@
+tags:  [monero, core]
+author: The Core Team
+---
+
+The year 2018 is well underway, with many exciting new technologies on the horizon for Monero. Technology is not the only thing that needs to adapt if Monero is to stay at the forefront of privacy and blockchain technologies, however. With that in mind, the Core Team would like to remind the community about their role in regards to Monero followed by an announcement regarding the membership of the Core Team. 
+
+The role of the Core Team can often be a confusing topic, especially with the various leadership structures employed by other cryptocurrencies. As the technology for a completely trustless society is not yet a reality, all projects, even decentralized ones, benefit from having individuals that can carry out tasks that require trust. A quick example of this is owning a domain name (like getmonero.org) or maintaining servers (on which getmonero.org is hosted). Ultimately, there is not yet a way for a domain name to be owned by a decentralized community, so someone has to be the name holding this domain. It's in scenarios like this where the Core Team are able to work on bridging the decentralized world we are trying to bring about with the centralized reality in which we are currently living.
+
+The responsibilities of the Core Team are as follows (in no particular order):
+    - Act as primary trusted arbiters of the Forum Funding System on behalf of the community, so as to ensure the completion of all projects to the satisfaction of the community.
+    - Manage the codebase of the Monero Project, which includes merging code on Github, keeping backups, and ensuring the safety, security, and free access of the code from any party.
+    - Steward the general donation fund, and spending the Monero there on anything they see fit to further the Monero Project.
+    - Act as trusted signers and distributors of reference clients for the Monero coin, and other related technologies.
+    - Set a direction and vision for the Monero Project
+    
+In the event that the community deems the Core Team unworthy or lax in their duties, they are free to fork the project away from the current Core Team, as happened with thankful_for_today.
+
+There are some things that the Core Team are NOT responsible for, despite misconceptions from the community. Some of the more common misconceptions are listed below:
+    - Members of the Core Team are NOT anyone's boss, and no permission is needed from them to do anything.
+    - The Core Team does NOT act as a centralized point of failure, but encourages organic, self-started initiatives that further the ecosystem of Monero.
+    - The Core Team does NOT equal Monero. In the event that one, or all of the Core Team goes rogue, we are to remember that Monero is a movement. A global initiative to further privacy globally, and provide real, fungible, digital money for everyone. This can happen even without the presence of the Core Team.

```

Comment:
>Monero is a movement

Monero is a software. Movements are centralized bodies which have leaders and funders - many of whom are working behind the scenes to spread their agenda. Monero is not a movement anymore than Windows is a movement (although Windows is a bowel movement, that's for sure).

>global initiative to further privacy globally

Implying that globalism is a requisite for providing privacy? That would be incorrect.

Monero can scale down to a small, local community and still serve it's purpose. Ironically, globalism is the antithesis of decentralization as it requires being built on centralized structures: both physically, socially, and metaphorically.

Reply:
> Monero is a movement

Monero is a software, but strongly ideals-driven. (Windows is not). Calling it a movement may be a strong word. But going to the other extreme, I would find limiting to define Monero as purely a software.